.github/workflows/ci.yml

```diff
@@ -24,112 +26,186 @@ jobs:
   build-dev:
     runs-on: mcf-dev-large-x64
     container: mobilecoin/fat-sgx-builder:v24.4.2
-
     steps:
-      - name: Check out code
-        uses: mobilecoinofficial/gh-actions/checkout@v0
-      - name: Cargo build (SW)
-        shell: bash
-        run: cargo build --locked
-      - name: Check dirty git
-        uses: ./.github/actions/check-dirty-git
+    - name: Check out code
+      uses: mobilecoinofficial/gh-actions/checkout@v0
+
+    - name: Cache Rust Binaries
```

Comment:
❓ This isn't actually caching the binaries is it?
My understanding is this is only caching the `success` sentinal file and using that to determin if this job needs to run

Reply:
There's no real point in caching/storing the binaries.  We are just marking the jobs as successful for this revision of the code.  If any of the rust files change, that will invalidate the cache and new tests will run.